lib/htmlnano.es6

```diff
@@ -37,6 +37,13 @@ export function loadConfig(options, preset, configPath) {
     ];
 }
 
+const optionalDependencies = {
+    minifyCss: ['cssnano', 'postcss'],
+    minifyJs: ['terser'],
+    minifyUrl: ['relateurl', 'srcset', 'terser'],
+    minifySvg: ['svgo'],
+};
```

Comment:
`purgecss` and `uncss` are not included here since only one of them is required when enabled.